op-supervisor/config/config.go

```diff
@@ -41,6 +42,9 @@ type Config struct {
 
 	Datadir             string
 	DatadirSyncEndpoint string
+
+	// OverrideMessageExpiryWindow overrides the protocol-defined message expiry window for testing
+	OverrideMessageExpiryWindow uint64
```

Comment:
I think this is a protocol parameter, and should be part of the interop dependency set definition. Just like the dependencies may be invalid before or after some time, the executing<>initiating distance may be configured in some way.

Adding it to the CLI and making it both a global and a chain-DB feature is really adding tech-debt, I am not confident it's the right long-term place for the parameter.

Reply:
My motivation for using the CLI parameter is to avoid pulling in the rollup config, which contains the expiry window override, into the op-supervisor.
Once upon a time, we went with placing the expiry window override in the rollup config so that the op-program can consume it without introducing yet another configuration. Today the op-program relies on the dependency set definition, similar to the op-supervisor. So the override can be moved into the depset config as you've suggested. Will do so.